README.md

```diff
@@ -1,17 +1,198 @@
-# Plastic SCM plugin for Jenkins
+<h1 align="center">
+    <img src="doc/img/logo-plasticscm.svg" alt="Plastic SCM Logo" width="450" />
+</h1>
 
-![Plugin Version](https://img.shields.io/jenkins/plugin/v/plasticscm-plugin.svg?label=version) [![Build Status](https://ci.jenkins.io/buildStatus/icon?job=Plugins/plasticscm-plugin/master)](https://ci.jenkins.io/job/Plugins/job/plasticscm-plugin/job/master/)
+<p align="center">
+    Retrieve and manage your Plastic SCM sources from Jenkins.
+</p>
 
-This plugin integrates [Jenkins](https://jenkins.io/) with [Plastic SCM](https://www.plasticscm.com/).
+---
+
+
+![Plugin Version](https://img.shields.io/jenkins/plugin/v/plasticscm-plugin.svg?label=version)
+[![Build Status](https://ci.jenkins.io/buildStatus/icon?job=Plugins/plasticscm-plugin/master)](https://ci.jenkins.io/job/Plugins/job/plasticscm-plugin/job/master/)
+![Installs](https://img.shields.io/jenkins/plugin/i/plasticscm-plugin.svg?color=blue)
+
+## Configuration
+### System configuration
+
+In order to configure the plugin, you need the `cm` executable installed in the Jenkins server
+machine. Then, please follow these steps:
+
+1. Open the system configuration page "Manage Jenkins" and navigate to "Configure System"
+2. Scroll down to the "Plastic SCM" section and enter the path where the `cm` executable (CLI
+   client) is located
+3. Click "Check" to verify that the path you set is correct
+4. You can leave the field empty: "`cm`" will be used by default (i.e. the executable will be
+   loaded from `$PATH`)
+
+![Manage Jenkins](doc/img/manage_jenkins.png)
+
+![Configure System](doc/img/configure_system.png)
+
+
+![Configure cm](doc/img/configuration-cm.png)
+
+## Job configuration
+### Freestyle projects
+
+Scroll down to the Source Code Management section and select "Plastic SCM" (see screenshot below).
+You'll need to enter a valid **selector**, which will be used to check out the repository contents
+in every build. The "Poll SCM" trigger also uses that selector.
+
+Every build will ensure that the Jenkins workspace contains a Plastic SCM before starting the
+`cm update` operation. If you'd like to create the Plastic SCM workspace in a particular
+subdirectory under the Jenkins job workspace, you can specify one in the **Subdirectory** field.
+This field is mandatory if you want to use multiple workspaces.
+
+If you want to keep the workspace between builds, check the **Use update box**. Otherwise, the
+Plastic SCM workspace and its contents will be deleted before each build.
+
+![Freestyle configuration](img/doc/freestyle-configuration.png)
```

Comment:
Looks like the link is broken

Reply:
Thank you @oleg-nenashev ! I mistyped the path, it should be `doc/img/freestyle-configuration.png`. I'll fix it right away!